charts/pravega-operator/templates/certificate.yaml

```diff
@@ -0,0 +1,50 @@
+apiVersion: certmanager.k8s.io/v1alpha1
+kind: Issuer
+metadata:
+  name: selfsigned-cert
+  namespace: {{ .Release.Namespace }}
+  labels:
+    app.kubernetes.io/name: pravega-operator
```

Comment:
shouldn't this be `{{- define "pravega-operator.fullname" -}}` instead of `pravega-operator`